### ISSUE TITLE
Applicability of the GeometryEngine

### ISSUE THREAD

**Issue by tulip411** [GeometryGym/GeometryGymIFC#28]
Hi there, 

I'm wondering if every building element can be parsed by the GeometryEngine. I was testing to read a building file in ifc format by using GeometryEngine, but some of the walls can not be found in the output geometry. I have no idea if it is my fault or some limities in GeometryEngine. 

Looking forward to your reply.

Sincerely
Maggie

**Comment by jmirtsch** [comment]
Hi Maggie, we support multiple schemas from ifc2x3 through to recent release candidate of 4x3.  
If you have problems with files, you can post them (or send privately) and I can take a look.  It could be a flaw in the file, or a bug in the toolkit, but hard to say without details.

Cheers,

Jon

**Comment by tulip411** [comment]
> Hi Maggie, we support multiple schemas from ifc2x3 through to recent release candidate of 4x3.
> If you have problems with files, you can post them (or send privately) and I can take a look. It could be a flaw in the file, or a bug in the toolkit, but hard to say without details.
> 
> Cheers,
> 
> Jon

Hi Jon, 

Thank you so much for your reply. It would be nice if I could send you the file privatly. May I send to this email: jonm@geometrygym.com ?

Sincerely
Maggie

**Comment by jmirtsch** [comment]
Yes, sharing the file privately is fine.  That email address is correct.
